src/mpp/Dec.hpp

```diff
@@ -327,18 +374,29 @@ struct Resolver {
 	}
 };
 
+/** Resolve path, semi-transparent object are visible. */
+template <size_t... P, class... T>
+constexpr auto&& path_resolve_transparent(tnt::iseq<P...>, T... t)
+{
+	using Res = Resolver<false, sizeof...(P) - 1, P...>;
+	static_assert(Res::expected_arg_count() == sizeof...(T));
+	return Res::get(t...);
+}
+
+/** Resolve path, semi-transparent object are invisible. */
```

Comment:
I guess the comments should be the other way around, i.e., visible for `path_resolve` and invisible for `path_resolve_transparent`.